### ISSUE TITLE
Mixed Support for Common Cartridge Discussion Topic Versions depending on how imported

### ISSUE THREAD

**Issue by CreepyGnome** [instructure/canvas-lms#1147]
When I import a pure common cartridge with resource type "imsdt_xmlv1p3" and schema "http://www.imsglobal.org/xsd/imsccv1p3/imsdt_v1p3" all of which is part of CC v1.3 it works fine. 

However your Canvas Importing of Canvas Exports using the canvas extensions/schema it requires that you use resource type "imsdt_xmlv1p1" and schema "http://www.imsglobal.org/xsd/imsccv1p1/imsdt_v1p1".

This seems very odd, from what I can you really have two different imports that can occur from the Common Cartridge import choice. If it has your resource "canvas_export.txt" it works as a flag to not do a CC import but to use only the canvas export files as the truth/source for the import. When it does this it seems to downgrade the supported versions of the Common Cartridge types and schemas that it still uses and relies on.

It seems that these two types of imports when it comes to Common Cartridge versions should support the same across the board. WIth them being fragmented it makes it hard to create a file that can work in any LMS that support CC v1.3 while if imported into Canvas LMS can take advantage of the extra features. This is because to do so we have to downgrade portions to CC 1.1 while keeping others 1.3. The parts you ignore we keep 1.3 and the ones you don't seem to have to be 1.1.

Please, I would like to see you keep both import paths when it comes to Common Cartridge version support to be the same.

**Comment by simonista** [comment]
Hey @CreepyGnome, thanks for the report. I agree that it's kinda sad that you can't take advantage of both CC v1.3 and canvas extra features in the same package. I think this is probably a fairly significant project to upgrade our internal canvas cartridge format to be based on CCv1.3 instead of CCv1.1. I'm going to reach out to the product manager over imports/exports and get their thoughts about it.

**Comment by CreepyGnome** [comment]
Until recently I figured you supported CCv1.3 as the UI and all docs stating you support 1.x it was only recently when we got into the weeds we found out your 1.x claim is not true as it is only 1.1.

If you end up not supporting CC 1.3, I would request that you change it from 1.x in the UI and docs to just be specific and say 1.1 so that it is clear and explicit. As stating 1.x is saying you support all versions of 1.0 and higher which isn't true.
